### DISCUSSION TITLE
Conteúdos da DB 193

### DISCUSSION THREAD

**Discussion by arturcorreaf** [YuriAlessandro/gerador-ficha-tormenta20#357] [Geral]
### Regra Opcional 

**Índole** - Sistema de pontos especiais que podem ser usados para aprimorar testes mal-sucedidos, recuperar pontos de mana e até mesmo evitar a morte! 

- Akzath
- Bravura
- Honra
- Instinto
- Lamento
- Liberdade
- Teimosia

### Ameaças

**Aberrações da Tormenta**

- Anomalocaris
- Falcão Fatídico
- Glop Sangrento
- Incubador
- Lursh-lyin
- Trilobita
- Nautilon

**Comment by YuriAlessandro** [comment]
Será adicionado

**Comment by YuriAlessandro** [comment]
Adicionado
